content/community/faq.md

```diff
@@ -45,28 +45,86 @@ Each release will fix known bugs and/or add new features.
 
 You can find the most recent version of Cucumber either in the [10-minute tutorial](/guides/10-minute-tutorial), the [installation](/installation) page or on [GitHub](https://github.com/cucumber).
 {{% block "java" %}}
-Note that with cucumber-jvm v2.x, the `groupId` has changed from `info.cukes` to `io.cucumber`.
+Note that with cucumber-JVM v2.x, the `groupId` has changed from `info.cukes` to `io.cucumber`.
```

Comment:
Should this name be first-letter capitalized?

Reply:
Yes, it should. Thanks for catching that!